lib/bolt/outputter/human.rb

```diff
@@ -312,7 +312,7 @@ def print_module_list(module_list)
       end
 
       def print_targets(options)
-        targets = options[:targets].map(&:name)
+        targets = options[:targets].map(&:safe_name)
```

Comment:
I'm assuming this change applies to  `bolt inventory show`? It makes more sense for `Target2`. Let me know if we want this or not. I'm leaning toward not doing it because it could.

Reply:
I think this should be the "string" representation of the target which I believe is the `name`

Reply:
I feel more strongly about the json output than this.